README.md

```diff
@@ -15,6 +15,14 @@ $ git clone https://github.com/SpiderOak/flow-python.git
 $ cd flow-python
 $ python setup.py install
 ```
+
+### Try it out with docker
+
+```
+$ docker build -t test/flow-python .
+$ docker run -it test/flow-python /usr/bin/python
```

Comment:
If this is the main use case I'd set an `ENTRYPOINT`, that way the users can do `docker run -it test/flow-python`. If there are other use cases maybe we want to use `CMD` instead.